src/zenml/cli/server.py

```diff
@@ -505,6 +505,32 @@ def deploy(
                 )
 
 
+# get outputs
+@cli.command(
+    "get-deploy-outputs",
+    help="Get the outputs of a ZenML server deployment.",
+)
+@click.argument(
+    "name",
+    required=True,
+    type=click.STRING,
+)
+@click.option(
+    "--output",
+    help="The name of the output to get.",
+    required=False,
+)
+def get_outputs(name: str, output: Optional[str] = None) -> Dict[str, str]:
```

Comment:
What if someone wants multiple outputs? I think you'll need a `multiple=True` here, and then a way to pass them into the `ServerDeployer` method to get multiple outputs.